### ISSUE TITLE
[Client] Add external theme file

### ISSUE THREAD

**Issue by rheimus** [superzanti/ServerSync#95]
Somewhat useless but cool none the less.

Can create a theme file that lets users change the color scheme for ServerSync.

**Comment by Hadyark** [comment]
Since v4 offers a new user interface that supports themes, maybe now something can be done

**Comment by rheimus** [comment]
Yeah could add some support for reading a custom theme config or something like that, I'll leave it as a backlog issue though as there are more pressing issues to solve around functionality and the current list of selectable themes is pretty good.
